HACKING.txt

```diff
@@ -1,15 +1,17 @@
 Hacking on Substance D
 ======================
 
-Here are some guidelines about hacking on Substance D.
+The following are guidelines and instructions for hacking on and with Substance
+D.
 
-Using a Development Checkout
-----------------------------
+Use a development checkout
+--------------------------
```

Comment:
- `Using` is consistent with `Adding` below.
- Why un-title-case section titles?